qutebrowser/browser/adblock.py

```diff
@@ -59,6 +61,24 @@ def get_fileobj(byte_io):
     return io.TextIOWrapper(byte_io, encoding='utf-8')
 
 
+def is_whitelisted_domain(host):
+    """Check if the given host is on the adblock whitelist.
+
+    Args:
+        host: The host as given by the adblocker as string.
+    """
+    whitelist = objreg.get('config').get('content', 'host-blocking-whitelist')
+    if whitelist is None:
+        return False
+
+    for domain in whitelist:
+        fnmatch_translated = fnmatch.translate(domain)
+        domain_regex = re.compile(fnmatch_translated, re.IGNORECASE)
+        if domain_regex.match(host):
```

Comment:
Was going the extra safe route here - if either `host` or `pattern/domain` contained uppercase characters - but it seems like `QUrl.host()` always returns lower-case.